app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
fix indentation, it will look better that way
```
def __init__(
        self,
        comfort_class: int,
        clean_mark: int,
        brand: str
) -> None:
```